force-app/main/default/lwc/errorPanel/errorPanel.js

```diff
@@ -1,26 +1,17 @@
 import { LightningElement, api, track } from 'lwc';
+import { reduceErrors } from 'c/ldsUtils';
 
 export default class ErrorPanel extends LightningElement {
     /** Generic / user-friendly message */
     @api friendlyMessage = 'Error retrieving data';
 
     @track viewDetails = false;
 
-    _errors;
+    /** Single or array of LDS errors */
+    @api errors;
```

Comment:
This prop is a non-primitive. Move transform logic to a getter used by the template so if the prop's value is mutated by the parent, a re-render is triggered and transformation logic runs on latest data.